bfm/README.md

```diff
@@ -0,0 +1,25 @@
+# Business File Manager (BFM)
+
+**Anmeldedaten für gemeinsames BFM-Konto**
+
+1. Zu dieser Seite gehen : [BFM](https://my.business-filemanager.de/Login.zul)
+2. Anmeldedaten : 
+
+- E-mail-Adresse : tractusxdays@gmail.com
+- Passwort : 3y)"59xvE16bhyJ]DfGl
+
+### **Screenshot-ID und Identifizierbarkeit:** 
+ 1. Vergabe einer Screenshot-ID
+   - Die ID soll so aussehen :
+ 
+       - ![alt text](image-1.png)
```

Comment:
Give a proper name to each artefact to be identifiable. 
ID should not be decided by the participant.